app/views/players/_form.html.slim

```diff
@@ -40,3 +40,11 @@
 			.form-group
 				label.d-block for="player_manual_seed" Manual Seed
 				=> f.text_field :manual_seed, placeholder: 'Set seed', class: 'manual_seed form-control'
+	- if defined?(new_registration) and not defined?(organiser_view)
+		= f.input :consent_data_sharing, as: :boolean,
+						class: 'form-check-input', input_html: { class: 'mr-1' },
+						inline_label: 'I consent for my personal data to be shared on this website'
```

Comment:
This should be more specific.  We have the NRDB username, when we do deck registration, we will have deck copies and we may have access to email from NRDB in the future, but we don't right now.

Maybe this could be a link to a new page for "what data is collected".

Reply:
As is, maybe change the text to 

I consent to share my data (including NRDB username and decks) with this site.

here and

I consent to share my decks with the tournament organizer.

Reply:
I've made it just one notice with more information.